Fix document not re-rendered & code cleanup

mistzzt opened the pull request:
This is a following up PR for #173.

* Fix inconsistent variable naming
* Move `OpenWithApplication` implementation out from `main.cpp`
* Add placeholder variables for `Info.plist`
* Fix the re-rendering issue

ahrm commented:
Thanks, but please don't increase the application version. We only do it every few months or so.
Also why are you feeding `file_name` to `handle_args` twice here?
```
	QObject::connect(&app, &OpenWithApplication::file_ready, [&main_widget](const QString& file_name) {
		main_widget.handle_args(QStringList() << file_name << file_name);
	});
```

mistzzt commented:
Thanks for the feedback. I've corrected the version.

Fyi, I added `VERSION = 1.2.0` in `pdf_viewer_build_config.pro` so that macOS can show the correct version information.

> Also why are you feeding file_name to handle_args twice here?

Correct me if I'm wrong: the first string in `args` is always the executable name, so it needs a placeholder here. Technically, it doesn't need to be `file_name`, but I'm unsure what kind of string literal I should use here for Qt.

It would be great if you could change it to something else that makes sense to you :)

ahrm commented:
I passed the application path as the first argument in 0cf30b167ba555d99e853b4542a3ac8d848fb4c1. Hopefully it doesn't break anything.
